cmd/operator-sdk/build/cmd.go

```diff
@@ -104,9 +99,8 @@ func buildFunc(cmd *cobra.Command, args []string) error {
 	}
 
 	image := args[0]
-	projutil.MustInProjectRoot()
 
-	if kbutil.HasProjectFile() {
+	if projutil.HasProjectFile() {
```

Comment:
It should be removed. All projects for 1.0.0 should have  `HasProjectFile `
The code inside of this checks is related to legacy layout.